mevislab.github.io/content/tutorials/openinventor.md

```diff
@@ -16,34 +16,39 @@ menu:
 # Open Inventor modules {#TutorialOpenInventorModules}
 ## Introduction
 
-Besides blue modules (ML modules) and brown Macro modules, there is a third type of modules, called Open Inventor modules. These modules are green and start with the letters `So\*` (for Scene Objects). Open Inventor modules process and render 3D scene objects and enable image interactions. Scene objects are transmitted via round input and output connectors. With the help of these modules, Open Inventor scenes can be implemented.
+In total, there are three types of modules:
+* Blue ML modules
+* Brown macro modules
+* Green Open Inventor modules
 
-This chapter will start with an example Open Inventor scene. Explanations to Open Inventor scenes will follow the example.
+The names of Open Inventor modules start with the prefix `So\*` (for Scene Objects). Open Inventor modules process and render 3D scene objects and enable image interactions. Scene objects are transmitted using the round input and output connectors. With the help of these modules, Open Inventor scenes can be implemented.
```

Comment:
round input and output connectors -> connectors represented as/shown as semicircles?